eng/pipelines/templates/steps/build-package-artifacts.yml

```diff
@@ -45,6 +45,11 @@ steps:
     displayName: 'Tag scheduled builds'
     condition: and(eq(variables['Build.SourceBranchName'], variables['DefaultBranch']), eq(variables['Build.Reason'],'Schedule'))
 
+  - ${{if eq(variables['System.TeamProject'], 'internal') }}:
```

Comment:
I'm unclear what adding this does for us? I assume the pipelines were working without it so what does adding the authentication to our dev feed buy us?

Reply:
1. CG runs execute properly on build jobs
2. Only using central feed for build, even if we have requirements pinned, is just a good thing to have.

We have this all for the `build_extended` and `test` jobs already, so keeping it around.

Any builds that fail as a result of missing upstream I'll monitor and resolve.